src/main/java/net/atos/zac/policy/PolicyService.java

```diff
@@ -85,18 +98,46 @@ public boolean isZaaktypeAllowed(final String zaaktypeIdentificatie) {
         return zaaktypeIdentificaties.contains(ALLE_ZAAKTYPEN) || zaaktypeIdentificaties.contains(zaaktypeIdentificatie);
     }
 
+    public static void assertActie(final boolean actie) {
+        if (!actie) {
+            throw new RuntimeException("Actie niet toegestaan");
```

Comment:
Hier misschien toch ook een nieuwe eigen exception voor gebruiken, zodat altijd duidelijk is bij het catchen dat ie hier vandaan kwam. PolicyException? En dan FoutmeldingException daar weer vanaf leiden?